pdc/apps/package/serializers.py

```diff
@@ -103,7 +103,7 @@ class Meta:
         fields = ('file_name', 'image_format', 'image_type', 'disc_number',
                   'disc_count', 'arch', 'mtime', 'size', 'bootable',
                   'implant_md5', 'volume_id', 'md5', 'sha1', 'sha256',
-                  'composes')
+                  'composes', 'subvariant')
```

Comment:
This filed is already in master branch. Adding this looks not necessary for fixing bug. Maybe it's for test?